app/controllers/users_controller.rb

```diff
@@ -116,8 +116,7 @@ def profile
                        .page(params[:page])
                        .order("nid DESC")
                        .where(status: 1, uid: @user.uid)
-    @questions = DrupalNode.questions
-                           .where(status: 1, uid: @user.uid)
+    @questions = @user.user.questions
```

Comment:
I'm so sorry, I believe I was mistaken that a double `user` was needed. This is a User record, not a DrupalUser record, so we should be fine just calling simply `@user.questions`. I believe that'll fix the test!